pyaerocom/stats/types.py

```diff
@@ -0,0 +1,17 @@
+from typing import Callable, Optional
+
+import numpy as np
+
+# Type definition for a callable which filters (ie. excludes) data before calculating stats.
+DataFilter = Callable[
```

Comment:
Would it make sense for a filter to instead return a mask of the (in)valid data? Saves some boilerplate for the implementor of the function